src/http-middlewares/before/prepare-request.js

```diff
@@ -101,8 +101,12 @@ const prepareRequest = function(req) {
 
   req = _.defaults(req, {
```

Comment:
In the future, we _could_ use `_.defaultsDeep` to be explicit about setting the defaults of a nested object like `removeMissingValuesFrom`.

But, since the defaults are false, if we only get something like:

```js
req: {
  removeMissingValuesFrom: {
    params: true,
  },
}
```

`_.defaults` will skip over `removeMissingValuesFrom` and `body` will be `undefined` which works for our default value of `false`.